Mathlib/Order/GameAdd.lean

```diff
@@ -19,16 +20,17 @@ This file defines, given relations `rα : α → α → Prop` and `rβ : β →
 decreasing either entry (with respect to `rα` and `rβ`). It is so called since it models the
 subsequency relation on the addition of combinatorial games.
 
+We also define `Sym2.GameAdd`, which is the unordered pair analog of `Prod.GameAdd`.
+
 ## Main definitions and results
 
 - `Prod.GameAdd`: the game addition relation on ordered pairs.
 - `WellFounded.prod_gameAdd`: formalizes induction on ordered pairs, where exactly one entry
   decreases at a time.
 
-## Todo
-
-- Add custom `induction` and `fix` lemmas.
-- Define `Sym2.GameAdd`.
+- `Sym2.GameAdd`: the game addition relation on unordered pairs.
+- `WellFounded.sym2_gameAdd`: formalizes induction on unordered pairs, where exactly one entry
+decreases at a time.
 -/
 
 
```

Comment:
In the mathport version the line below this one uses `{}`

Reply:
Both review comments addressed, @semorrison